internal/pkg/template/workload.go

```diff
@@ -215,6 +216,17 @@ type Topics struct {
 	AllowedWorkers []string
 }
 
+// SubscribeOpts holds configuration needed if the service has subscriptions.
+type SubscribeOpts struct {
+	Topics []*TopicSubscription
+}
+
+// TopicSubscription holds information needed to render a SNS Topic Subscription in a container definition.
+type TopicSubscription struct {
+	Name    *string
```

Comment:
That makes sense!

I agree that we are not super consistent on when a pointer type should be used :(  In this case, I feel like since `Name` is a required field for a topic anyway, it doesn't need to be a pointer. 

I'm good with keeping it as a pointer though - it doesn't hurt since go template dereferences a pointer while parsing.